restAPI_ecommerce/src/main/java/e_commerce/restAPI_ecommerce/services/springdatajpa/OwnerJPAService.java

```diff
@@ -0,0 +1,6 @@
+package e_commerce.restAPI_ecommerce.services.springdatajpa;
+
+import e_commerce.restAPI_ecommerce.services.OwnerService;
+
+public class OwnerJPAService implements OwnerService {
```

Comment:
rename it to OwnerServiceImpl, don't add language/framework specific keywords like JPA